code/src/pages/NotFound.css

```diff
@@ -0,0 +1,31 @@
+.not-found-wrapper {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: 10px;
+}
+
+.not-found-wrapper h2 {
```

Comment:
I like that you haven't done classes for all these "h2" and "p" etc - and uses the css file this way instead (if that was the purpose of it). I think it makes the code more clean and it's easy to grasp where to find each element instead of transforming them into a className that could be hard to remember and expands the code in the js file.